<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>443</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Movie Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="game_tab">
       <attribute name="title">
        <string>Game Properties</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_16">
        <item row="0" column="0">
         <widget class="QScrollArea" name="gamePropertiesScrollarea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="GamePropertiesScrollareaContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>828</width>
             <height>275</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Game Scenes</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QScrollArea" name="gameScenes_scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_8">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>828</width>
             <height>275</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="stage_tab">
       <attribute name="title">
        <string>Stages</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QScrollArea" name="StagesScrollarea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>828</width>
             <height>275</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="maps_tab">
       <attribute name="title">
        <string>Maps</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_17">
        <item row="0" column="0">
         <widget class="QScrollArea" name="MapScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>828</width>
             <height>275</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="anim_blocks_tab">
       <attribute name="title">
        <string>Anim Tiles</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QScrollArea" name="anim_tab_scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAnimContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>828</width>
             <height>275</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="weapon_edit_tab">
       <attribute name="title">
        <string>Weapons</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QScrollArea" name="WeaponScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>828</width>
             <height>275</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_projectiles">
       <attribute name="title">
        <string>Projectiles</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QScrollArea" name="ProjectileScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_4">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>828</width>
             <height>275</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="npc_tab">
       <attribute name="title">
        <string>Enemies</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_11">
        <item row="0" column="0">
         <widget class="QScrollArea" name="NPCScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_5">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>828</width>
             <height>275</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ai_tab">
       <attribute name="title">
        <string>Artificial Inteligence</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_12">
        <item row="0" column="0">
         <widget class="QScrollArea" name="AIScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_6">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>828</width>
             <height>275</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="object_tab">
       <attribute name="title">
        <string>Objects</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_14">
        <item row="0" column="0">
         <widget class="QScrollArea" name="ObjectScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_7">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>828</width>
             <height>275</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="player_tab">
       <attribute name="title">
        <string>Player Edit</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_18">
        <item row="0" column="0">
         <widget class="QScrollArea" name="PlayerScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_9">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>828</width>
             <height>275</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="armor_tab">
       <attribute name="title">
        <string>Armor Pieces</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QScrollArea" name="armorScrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_10">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>828</width>
             <height>275</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionMovie_Editor"/>
   <addaction name="actionStrings_Editor"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFILE">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionRun_Stage_2"/>
    <addaction name="actionSwap_Maps"/>
    <addaction name="actionReset_Stage_Links"/>
    <addaction name="actionReset_Map"/>
    <addaction name="separator"/>
    <addaction name="actionScenes_Editor"/>
    <addaction name="actionImage_Browser"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuLevel">
     <property name="title">
      <string>Level</string>
     </property>
     <addaction name="actionOne"/>
     <addaction name="actionTwo"/>
     <addaction name="actionThree"/>
    </widget>
    <widget class="QMenu" name="menuZoom">
     <property name="title">
      <string>Zoom</string>
     </property>
     <addaction name="actionZoomOne"/>
     <addaction name="actionZoomTwo"/>
     <addaction name="actionZoomThree"/>
    </widget>
    <addaction name="menuLevel"/>
    <addaction name="menuZoom"/>
    <addaction name="actionGRID"/>
    <addaction name="actionObjects"/>
    <addaction name="actionNPCs"/>
    <addaction name="actionTeleporters"/>
   </widget>
   <addaction name="menuFILE"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/toolbar_icons/New</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New Game</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/Open</normaloff>
     <normalon>:/toolbar_icons/Save</normalon>:/toolbar_icons/Open</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/Save</normaloff>:/toolbar_icons/Save</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/list-add.png</normaloff>:/toolbar_icons/list-add.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionLayer_Level">
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/format-list-ordered.png</normaloff>:/toolbar_icons/format-list-ordered.png</iconset>
   </property>
   <property name="text">
    <string>Layer Level</string>
   </property>
   <property name="toolTip">
    <string>Layer Level</string>
   </property>
  </action>
  <action name="actionOne">
   <property name="text">
    <string>one</string>
   </property>
  </action>
  <action name="actionTwo">
   <property name="text">
    <string>two</string>
   </property>
  </action>
  <action name="actionThree">
   <property name="text">
    <string>three</string>
   </property>
  </action>
  <action name="actionReset_Map">
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/archive-remove.png</normaloff>:/toolbar_icons/archive-remove.png</iconset>
   </property>
   <property name="text">
    <string>Reset Map</string>
   </property>
   <property name="toolTip">
    <string>Clear all map tiles</string>
   </property>
  </action>
  <action name="actionSwap_Maps">
   <property name="text">
    <string>Swap Stages</string>
   </property>
  </action>
  <action name="actionScenes_Editor">
   <property name="text">
    <string>Movie Editor</string>
   </property>
  </action>
  <action name="actionObjects">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Objects</string>
   </property>
  </action>
  <action name="actionNPCs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enemies</string>
   </property>
  </action>
  <action name="actionTeleporters">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Teleporters</string>
   </property>
  </action>
  <action name="actionImage_Browser">
   <property name="text">
    <string>Image Browser</string>
   </property>
  </action>
  <action name="actionMovie_Editor">
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/insert-image.png</normaloff>:/toolbar_icons/insert-image.png</iconset>
   </property>
   <property name="text">
    <string>Movie Editor</string>
   </property>
  </action>
  <action name="actionStrings_Editor">
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/draw-text.png</normaloff>:/toolbar_icons/draw-text.png</iconset>
   </property>
   <property name="text">
    <string>Strings Editor</string>
   </property>
  </action>
  <action name="actionReset_Stage_Links">
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/dialog-warning.png</normaloff>:/toolbar_icons/dialog-warning.png</iconset>
   </property>
   <property name="text">
    <string>Reset Stage Links</string>
   </property>
   <property name="toolTip">
    <string>Remove All Map Links</string>
   </property>
  </action>
  <action name="actionZoomOne">
   <property name="text">
    <string>One</string>
   </property>
  </action>
  <action name="actionZoomTwo">
   <property name="text">
    <string>Two</string>
   </property>
  </action>
  <action name="actionZoomThree">
   <property name="text">
    <string>Three</string>
   </property>
  </action>
  <action name="actionGRID">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>gRID</string>
   </property>
  </action>
  <action name="actionRun_Stage_2">
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/toolbar_icons/arrow-right.png</normaloff>:/toolbar_icons/arrow-right.png</iconset>
   </property>
   <property name="text">
    <string>Run Stage</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources/icons/icons.qrc"/>
  <include location="resources/icons/icons.qrc"/>
  <include location="resources/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
